Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -245,14 +249,22 @@ public class HTTPServer : Server {
         return server
     }
     
-    deinit { 
-        try! eventLoopGroup.syncShutdownGracefully()
+    deinit {
+        do {
+            try eventLoopGroup.syncShutdownGracefully()
+        } catch {
+            Log.error("Failed to shutdown eventLoopGroup")
+        }
     }
 
     /// Stop listening for new connections.
     public func stop() {
         guard serverChannel != nil else { return }
-        try! serverChannel.close().wait()
+        do {
+            try serverChannel.close().wait()
+        } catch let error {
+            Log.error("Error: \(error)")
```

Comment:
Can we have a more detailed error, something like `Failed to close the server channel. Error: \(error)`